specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1589,7 +1589,7 @@
       "properties": {
         "exportable": {
           "type": "boolean",
-          "description": "Indicates if the private key can be exported."
+          "description": "Indicates if the private key can be exported. Release policy must be provided when creating 1st version of exportable key."
```

Comment:
Release policy must be provided when creating THE 1st version of AN exportable key.

Reply:
Fixed